js/cryptocom.js

```diff
@@ -574,6 +575,77 @@ module.exports = class cryptocom extends Exchange {
         return result;
     }
 
+    async fetchLastPrices (symbols = undefined, params = {}) {
+        await this.loadMarkets ();
+        const [ marketType, query ] = this.handleMarketTypeAndParams ('fetchLastPrices', undefined, params);
```

Comment:
@ttodua hum this is not working anymore, can you take a look?

```
n cryptocom fetchLastPrices
2023-02-13T15:13:31.047Z
Node.js: v17.9.0
CCXT v2.7.92
cryptocom.fetchLastPrices ()
---------------------------------------------------
[TypeError] this[method] is not a function

    at fetchLastPrices            js/cryptocom.js:591                   const response = await this[method] (query);        
    at processTicksAndRejections  node:internal/process/task_queues:96                                                      
    at async run                  examples/js/cli.js:293                const result = await exchange[methodName] (... args)
```

Reply:
@ttodua I'm going to remove this implementation for now, this `getTrades` does not seem the right endpoint to do this. Feel free to re-add it later

Reply:
seems they have removed from v2 api. so, we can ignore cryptocom.